src/test/java/org/apache/commons/imaging/ImageDumpTest.java

```diff
@@ -0,0 +1,44 @@
+package org.apache.commons.imaging;
+
+import org.junit.Test;
+
+import java.awt.color.ColorSpace;
+import java.awt.image.BufferedImage;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Unit tests for class {@link ImageDump}.
+ *
+ * @date 2017-07-13
+ * @see ImageDump
+ *
+ **/
```

Comment:
I question the presence of Javadoc in this class and all the other classes affected by this PR. AFAICT not all the tests (if any) even have Javadocs, so why should javadocs be added for this class and the others?